.github/workflows/e2etest-servicemeshandaddon.yaml

```diff
@@ -93,7 +93,7 @@ jobs:
           echo $pretty > data.json
           cat data.json
           touch test.md
-          filename=$(date '+%Y-%m-%d-%H:%M:%S')
+          filename=$(date '+%Y-%m-%d_%H-%M-%S')
```

Comment:
@leecalcote is this format okay for the timestamp?

Reply:
Proposal: don't capture time in the filename. Use this approach - https://github.com/meshery/meshery/pull/4820#issuecomment-1000349639

Reply:
I edited the content now... Maybe now it'll work fine?